Mathlib/GroupTheory/SemidirectProduct.lean

```diff
@@ -305,4 +311,8 @@ def congr' :
 
 end Congr
 
+@[simp]
+lemma card : Nat.card (N ⋊[φ] G) = Nat.card N * Nat.card G :=
+  Nat.card_prod _ _ ▸ Nat.card_eq_of_bijective _ equivProd.bijective
```

Comment:
Optional, but this might be nicer with [Nat.card_congr](https://leanprover-community.github.io/mathlib4_docs/Mathlib/SetTheory/Cardinal/Finite.html#Nat.card_congr) instead of `Nat.card_eq_of_bijective`.